scripts/variantstore/wdl/extract/uber_monitor.py

```diff
@@ -0,0 +1,186 @@
+import argparse
+import sys
+import re
+import os
+
+global MaxCpu
+global MaxMem
+global MaxMemPct
+global MaxDisk
+global MaxDiskPct
```

Comment:
I don't think these need to be declared `global` here

Reply:
Got rid of those